packages/next/client/image.tsx

```diff
@@ -99,8 +101,12 @@ type CallLoaderProps = {
   width: number
   quality?: number
 }
+type URLResolver = (resolverProps: CallLoaderProps) => string
 
-function callLoader(loaderProps: CallLoaderProps) {
+function callLoader(loaderProps: CallLoaderProps, loader?: URLResolver) {
```

Comment:
```suggestion
function callLoader(loaderProps: CallLoaderProps, loader: URLResolver | undefined) {
```

This will make sure you don't forget the param